src/files/files.go

```diff
@@ -0,0 +1,341 @@
+	startPosition := strings.Index(messageRow, messageTag) + len(messageTag)
+
+	if startPosition-1 == len(messageRow)-1 {
+		return "", errors.New("message not found")
+	}
+
+	message := strings.TrimSpace(messageRow[startPosition:])
+	if strings.ReplaceAll(message, " ", "") == "" {
+		return "", errors.New("message not found")
+	}
+
+	return message, nil
+}
+
+// pretiffy aims to keep a short message based on the commit message, removing extra information such as commit type.
+// Args:
+// 		commitMessage (string): Full commit message.
+// Returns:
+// 		string: Returns a commit message with limmited number of characters.
+// 		err: Error whenever unexpected issues happen.
+func (f *FileVersion) prettifyCommitMessage(commitMessage string) (string, error) {
+
+	var message string
+	splitedMessage := strings.Split(commitMessage, "\n")
+
+	for _, row := range splitedMessage {
+		row := strings.ToLower(row)
+		if f.findMessageTag(row) {
+
+			currentMessage, err := f.getMessage(row)
+			if err != nil {
+				return "", err
+			}
+			message = currentMessage
+		}
+	}
+
+	if message == "" {
+		return "", errors.New("commit message has no tag 'message:'")
+	}
+	// Limmit message to 150 characters to avoid long messages.
+	if len(message) >= 150 {
+		message = fmt.Sprintf("%s...", message[:150])
+	}
+
+	// Upper only first letter of the sentence.
+	message = fmt.Sprintf("%s%s", strings.ToUpper(message[:1]), message[1:])
+	return message, nil
+}
+
+func (f *FileVersion) containsVariableNameInText(text, variableName, copySignal string) bool {
+	text = strings.ReplaceAll(text, " ", "")
+	return strings.Contains(text, fmt.Sprintf("%s%s", variableName, copySignal))
+}
+
+func (f *FileVersion) setDefaultPath(path, newPath string) string {
+	if path != "" {
+		return path
+	}
+	return newPath
+}
+
+func (f *FileVersion) unmarshalUpgradeFiles(filesToUpgrade interface{}) (*UpgradeFiles, error) {
+	filesToUpdateBytes, err := json.Marshal(filesToUpgrade)
+	if err != nil {
+		return nil, errors.New("error marshalling files to uptade")
+	}
+
+	var filesToUpdateResult UpgradeFiles
+	if err := json.Unmarshal(filesToUpdateBytes, &filesToUpdateResult); err != nil {
+		return nil, errors.New("error unmarshalling files to uptade")
+	}
+
+	return &filesToUpdateResult, nil
+}
+
+// UpgradeVariableInFiles aims to update given files with the new release version.
+// It will update the files row containing a given variable name.
+// I.e.:
+// err := UpgradeVariableInFiles(UpgradeFiles{Files: []UpgradeFile{{Path: "./setup.py", DestinationPath: "", VariableName: "__version__"}}), "1.0.1")
+//  	From: __version__ = 1.0.0
+//  	To:   __version__ = 1.0.1
+func (f *FileVersion) UpgradeVariableInFiles(filesToUpgrade interface{}, newVersion string) error {
+	defer f.elapsedTime("UpgradeVariableInFiles")()
+
+	var outputData []byte
+
+	filesToUpdate, err := f.unmarshalUpgradeFiles(filesToUpgrade)
+	if err != nil {
+		return err
```

Comment:
a lot of errors without context... errors like EOF... don't help much...but if we inform where the error happened.. maybe it will help more... like... "error on unmarshal file / error: EOF"

@esequielvirtuoso

Reply:
Done.